src/Compilers/CSharp/CscCore/CscCore.csproj

```diff
@@ -52,6 +52,26 @@
     <PlatformTarget>x64</PlatformTarget>
     <ErrorReport>prompt</ErrorReport>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <DebugSymbols>true</DebugSymbols>
```

Comment:
Many of these properties are incorrect. Also, I don't think we need to add a configuration, but change the existing configuration to be AnyCPU.